src/main/java/org/fenixedu/academic/ui/struts/action/internationalRelatOffice/candidacy/erasmus/ErasmusIndividualCandidacyProcessDA.java

```diff
@@ -392,6 +392,9 @@ public ActionForward doEnrol(ActionMapping mapping, ActionForm form, HttpServlet
             addActionMessage("error", request, ex.getKey(), ex.getArgs());
 
             return enrolStudent(mapping, form, request, response);
+        } catch (org.fenixedu.bennu.core.domain.exceptions.DomainException de) {
```

Comment:
The other place in this class where a DomainException is used could be replaced by this type